pkg/network/protocols/http/monitor.go

```diff
@@ -152,14 +153,15 @@ func NewMonitor(c *config.Config, offsets []manager.ConstantEditor, sockFD *ebpf
 	state = Running
 
 	httpMonitor := &Monitor{
-		ebpfProgram:     mgr,
-		httpTelemetry:   httpTelemetry,
-		http2Telemetry:  http2Telemetry,
-		closeFilterFn:   closeFilterFn,
-		statkeeper:      statkeeper,
-		processMonitor:  processMonitor,
-		http2Enabled:    c.EnableHTTP2Monitoring,
-		http2Statkeeper: http2Statkeeper,
+		ebpfProgram:        mgr,
+		httpTelemetry:      httpTelemetry,
+		http2Telemetry:     http2Telemetry,
+		closeFilterFn:      closeFilterFn,
+		statkeeper:         statkeeper,
+		processMonitor:     processMonitor,
+		http2Enabled:       c.EnableHTTP2Monitoring,
+		http2Statkeeper:    http2Statkeeper,
+		javaOrGoTLSEnabled: c.EnableJavaTLSSupport || c.EnableGoTLSSupport,
```

Comment:
If this isn't being used elsewhere, I'd just make the `if`-statement below use `c.EnableJavaTLSSupport || c.EnableGoTLSSupport`